adcircpy/forcing/winds/best_track.py

```diff
@@ -119,22 +119,20 @@ def summary(self, output: Union[str, PathLike] = None):
         track_length = self.track_length
         duration = self.duration
         min_central_pressure = np.min(self.central_pressure)
-        max_wind_speed = np.max(self.df["max_sustained_wind_speed"])
-        start_loc = (self.df["longitude"][0], self.df["latitude"][0])
-        end_loc = (self.df["longitude"].iloc[-1], self.df["latitude"].iloc[-1])
+        max_wind_speed = np.max(self.df['max_sustained_wind_speed'])
+        start_loc = (self.df['longitude'][0], self.df['latitude'][0])
+        end_loc = (self.df['longitude'].iloc[-1], self.df['latitude'].iloc[-1])
         if output is None:
-            _storm = f"Summary of storm: {self.storm_id}"
+            _storm = f'Summary of storm: {self.storm_id}'
             print(_storm.upper())
-            print(
-                f"min./max. track speed: {min_storm_speed} m/s, {max_storm_speed} m/s"
-            )
-            print(f"min. central pressure: {min_central_pressure} hPa")
-            print(f"max. wind speed: {max_wind_speed} kts")
-            print(f"Starting at: {start_loc} and ended at: {end_loc}")
-            print(f"Total track length: {track_length:.2f} km")
-            print(f"Total track duration: {duration:.2f} days")
+            print(f'min./max. track speed: {min_storm_speed} m/s, {max_storm_speed} m/s')
+            print(f'min. central pressure: {min_central_pressure} hPa')
+            print(f'max. wind speed: {max_wind_speed} kts')
+            print(f'Starting at: {start_loc} and ended at: {end_loc}')
+            print(f'Total track length: {track_length:.2f} km')
+            print(f'Total track duration: {duration:.2f} days')
         else:
-            print("not completed")
+            print('not completed')
```

Comment:
I think in cases like this it's more appropriate to use `raise NotImplementedError("Brief explanation of what's missing")`.

Reply:
Okay. I'm not sure yet how to I might use this information yet so that's why I haven't implemented it.